.github/workflows/build.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             INTERFACE64: 1
       fail-fast: false
     runs-on: ${{ matrix.os }}
+    env:
+      ANACONDA_SCIENTIFIC_PYTHON_UPLOAD: ${{ secrets.ANACONDA_SCIENTIFIC_PYTHON_UPLOAD }}
```

Comment:
This should not be in the global environment, rather in the Upload-specific environment below

Reply:
Completely missed that :facepalm: